src/main/java/touro/peg/LegalMovesFactory.java

```diff
@@ -10,7 +9,7 @@ public LegalMovesFactory() {
         this.legalMoves = getLegalMovesList();
     }
 
-    private List<Move> getLegalMovesList() {
+    List<Move> getLegalMovesList() {
```

Comment:
This should remain private.
You can retrieve the list by getting the `legalMoves` value in here.